docs/_includes/jlbp-header.html

```diff
@@ -0,0 +1,61 @@
+<!-- prettier-ignore -->
+<header class="glue-header glue-header--single">
+  <div class="glue-header__bar glue-header__bar--desktop glue-header__drawer">
+    <div class="glue-header__tier">
+      <!-- LOCK up-->
+      <div class="glue-header__container">
+        <div class="glue-header__lock-up">
+          <div class="glue-header__logo">
+            <a class="glue-header__logo-link" href="{{ '/' | relative_url }}" title="Google">
+              <div class="glue-header__logo-container">
+                <div class="cloud-logo">
+                  <img src="/assets/images/Cloud_Logo_Nav.svg" alt="Google Cloud">
+                </div>
+              </div>
+              <p class="glue-header__logo--product">JLBPs</p>
+            </a>
+          </div>
+          <a href="#page-content" class="glue-header__skip-content">
+            <p class="glue-header__skip-content-text">Jump to Content</p>
+          </a>
+        </div>
+      </div>
+      <!-- LINK BAR-->
+      <div class="glue-header__container glue-header__container--flex-auto">
+        <nav class="glue-header__link-bar">
+        </nav>
+      </div>
+    </div>
+  </div>
+  <!-- Mobile header placeholder -->
+  <div class="glue-header__bar glue-header__bar--mobile">
+    <div class="glue-header__tier">
+      <!-- LOCK up-->
+      <div class="glue-header__container">
+        <div class="glue-header__lock-up">
+          <div class="glue-header__hamburger glue-header__hamburger--first-tier">
```

Comment:
This doesn't seem to show up and it issues an accessibility warning:
"Clickable elements must be focusable and should have interactive semantics. Learn more"

Can we delete it?

Reply:
This is taken straight from the Glue sample:

https://glue-docs.appspot.com/docs/components/raw/header#HTML

Also it only appears on mobile mode. Shrink the window down to see it appear.